hyp3lib/simplify_shapefile.py

```diff
@@ -62,7 +59,7 @@ def simplify_shapefile(inshp,outshp):
         # post a request for simplification service
         try:
             response = requests.post('https://api.daac.asf.alaska.edu/services/utils/files_to_wkt', files=files)
-        except:
+        except requests.RequestException:
             logging.error("ERROR: service unavaible - it may be that your shapefile is too large.  Reduce to under 300 points")
```

Comment:
The error handling here can be improved, probably a single `response.raise_for_status()` call, maybe with a check for the specific message or status code that api.daac.asf.alaska.edu throws in response to a bad shape file.  Fine punting it for now.

Reply:
Closing this, read the whole function and it's more work than a quick band-aid.